### DISCUSSION TITLE
Me being stupid

### DISCUSSION THREAD

**Discussion by fethomm** [Kovah/LinkAce#531] [Support] [answered]
I have installed LinkAce lately with Docker compose in Proxmox and just did an upgrade. Tried logging back in afterwards and found out, that I had not noted the email address and password to login Trying to reset the password. LinkAce tells me that none of the possibly used email addresses matches any user. Is there a way to find out the email address I used?

**Comment by Kovah** [comment]
Do you have access to the database? There are some tools you can use to connect, with the credentials from the .env file. 
The users table contains all users.

If you have made a backup of the database, you should find the user details in that file.

**Reply by fethomm** [reply]
I have access to the file system in konsole of Proxmox. Where would I look?

**Reply by Kovah** [reply]
Does that mean you have made a backup of the database? I think you would have to unpack/unzip the backup stored in the `backups` folder, and have a look in the database backup (not sure what it is called, but should end in .sql or something like that).

**Comment by fethomm** [comment]
I have Proxmox backup running, so yes, I have a backup of the VM. Will look into that. Thanks so far :)

**Reply by fethomm** [reply]
I found the .env, but unfortunately, it does not have information on where the db is located. And for the life of me, I can't find it. I am quite new to Docker, so I don't know my way around. I am in the middle of writing an article on LinkAce for German print magazine LinuxUser and I am totally stuck right now.

**Reply by Kovah** [reply]
Feel free to contact me via contact [at] linkace [dot] org for detailed troubleshooting if you want. We for sure can solve this issue.

**Comment by fethomm** [comment, marked as answer]
I was able to extract the links from the database with the help of some Regex. Thanks for your help.
